fbs/matrix.fbs

```diff
@@ -62,6 +63,17 @@ table DictEncoded32FBArray {
   dict: [uint8];
 }
 
+table Int16EncodedXFBArray {
+  // the integer codes corresponding to float bins
+  codes: [int16];
+  // the maximum value of the source array
+  max: float32;
+  // the minimum value of the source array
+  min: float32;
+  // the number of bins
+  nbins: int32;
+}
+
```

Comment:
worth spelling out here how the encoding works, maybe just via a single math expression (repeating [this](https://github.com/chanzuckerberg/single-cell-explorer/pull/437/files#diff-0c1e70ff198b19b032eaf0c7e925217a2bfa6e135bdd6eab94f095bdcfd6b2bfR52))